newroutes/netlifyToml.js

```diff
@@ -0,0 +1,36 @@
+const autoBind = require("auto-bind")
+const express = require("express")
+
+const { attachReadRouteHandlerWrapper } = require("@middleware/routeHandler")
+
+class NetlifyTomlRouter {
+  constructor({ netlifyTomlService }) {
+    this.netlifyTomlService = netlifyTomlService
+    // We need to bind all methods because we don't invoke them from the class directly
+    autoBind(this)
+  }
+
+  // Read netlify.toml file
+  async readNetlifyToml(req, res) {
+    const { accessToken } = req
+
+    const netlifyTomlHeaderValues = await this.netlifyTomlService.read({
+      accessToken,
+    })
+
+    return res.status(200).json({ netlifyTomlHeaderValues })
+  }
+
+  getRouter() {
+    const router = express.Router()
+
+    router.get(
+      "/netlify-toml",
```

Comment:
we should really maintain a mapping of keys to raw values

Reply:
Could you clarify which raw values you're referring to?

Reply:
the `/netlify-toml`. idt it's a big issue cos the path should be used minimally? (lmk if i'm wrong)

Reply:
Right, this is only used for this single endpoint - do you think router namespaces will help with this issue?